packages/algob/src/lib/compile.ts

```diff
@@ -117,13 +121,16 @@ export class PyCompileOp {
    *                   Examples : [ gold.py, asa.py]
    *                   MUST have .py extension
    * @param force    : if true it will force recompilation even if the cache is up to date.
+   * @param scInitParam : Smart Contract Initialization Parameters
    */
-  async ensureCompiled (filename: string, force: boolean): Promise<PyASCCache> {
+  async ensureCompiled (filename: string, force?: boolean, scInitParam?: Object): Promise<PyASCCache> {
```

Comment:
Please update the docstring to explain parameters.

Reply:
ok